packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -84,15 +90,18 @@ export function TransferPanel() {
     useState<TokenDepositCheckDialogType>('new-token')
   const [importTokenModalStatus, setImportTokenModalStatus] =
     useState<ImportTokenModalStatus>(ImportTokenModalStatus.IDLE)
+  const [showSCWalletTooltip, setShowSCWalletTooltip] = useState(false)
 
   const {
     app: {
       connectionState,
+      walletType,
       selectedToken,
       isDepositMode,
       arbTokenBridgeLoaded,
       arbTokenBridge: { eth, token, walletAddress },
       arbTokenBridge,
+      transactionSettings,
```

Comment:
I might be missing something, but why do we lift this all the way up to the global store instead of just having it as a local state in `TransferPanel`?

Reply:
@spsjvc the idea here was to keep transactionSettings scalable, also it is already user in 2 places so I wanted to avoid drilling props. Although if we are not concerned about scaling advanced settings in the future we could keep it in the local state

Reply:
I'd keep it in the local state to avoid cluttering global state. If we ever decide to move it to global state, it's easier than the other way around, I think.